a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -761,6 +763,92 @@ <h4>Display techniques for support for nonvisual reading support
 				</ul>
 			</section>
 		</section>
+
+		<section id="prerecorded-audio">
+			<h4 data-localization-id="prerecorded-audio-title">Prerecorded audio</h4>
```

Comment:
ID should be "ways-of-reading-prerecorded-audio-title"
to mirror what we did for "additional-accessibility-information-clarity-title"